davinci-resolve-checker.py

```diff
@@ -36,7 +36,7 @@
 # lspci -d ::0380 - amd secondary gpu on an i+a laptop
 
 print("Presented GPUs:")
-print ("\t" + "\n\t".join([ x.device.name + " (kernel driver in use: " + x.driver + ")" for x in lspci_devices if x.cls.id in (0x0300, 0x0301, 0x0302, 0x0380) ]))
+print ("\t" + "\n\t".join([ str(x.device.name) + " (kernel driver in use: " + str(x.driver) + ")" for x in lspci_devices if x.cls.id in (0x0300, 0x0301, 0x0302, 0x0380) ]))
```

Comment:
Why this change? x.device.name and x.driver are already str. Had you another case?

Reply:
Yes there was an error in printing those 2 without  convering them with  str

Reply:
Can you please provide more details? Which devices do you have? Do the error appears at only specific device? What type is it before explicitly converting to str? Does in lspci output there is a driver assigned?
I am interested because I just run it without convertion, and it works ok for me.

Reply:
OS: Garuda Linux 
Host: 80TV Lenovo ideapad 310-15IKB 
Kernel: 5.10.69-1-lts 
DE: Plasma 5.22.5 
WM: KWin 
CPU: Intel i7-7500U (4) @ 3.5GHz 
GPU: Intel HD Graphics 620 
GPU: NVIDIA GeForce 920MX 
Memory: 3.24GiB / 7.67GiB 

when I run prime-run ./davinci-resolve-checker.py

it gives the output : 

DaVinci Resolve checker 1.6.2
Installed DaVinci Resolve package:
Chassis type: laptop
Installed OpenCL drivers: intel-compute-runtime lib32-opencl-nvidia opencl-nvidia
Presented GPUs:
Traceback (most recent call last):
File "/home/egyleader/dev/davinci-resolve-checker/./davinci-resolve-checker.py", line 39, in <module>
print ("\t" + "\n\t".join([ x.device.name + " (kernel driver in use: " + x.driver + ")" for x in lspci_devices if x.cls.id in (0x0300, 0x0301, 0x0302, 0x0380) ]))
File "/home/egyleader/dev/davinci-resolve-checker/./davinci-resolve-checker.py", line 39, in <listcomp>
print ("\t" + "\n\t".join([ x.device.name + " (kernel driver in use: " + x.driver + ")" for x in lspci_devices if x.cls.id in (0x0300, 0x0301, 0x0302, 0x0380) ]))
TypeError: can only concatenate str (not "NoneType") to str